docs/index.rst

```diff
@@ -0,0 +1,35 @@
+.. toctree::
+   For Enterprise <https://tidelift.com/subscription/pkg/pypi-|project|?utm_source=pypi-|project|&utm_medium=referral&utm_campaign=docs>
```

Comment:
When there's no space before the first pipe, the RST parser does not recognize that as a substitution. Use the hack with an escaped whitespace to fix that. But I'm not sure if the toctree directive renders RST args so this needs to be tried out.
```suggestion
   For Enterprise <https://tidelift.com/subscription/pkg/pypi-\ |project|?utm_source=pypi-\ |project|&utm_medium=referral&utm_campaign=docs>
```

Reply:
Unfortunately, that doesn't seem to work. The substitution still doesn't take:

![image](https://user-images.githubusercontent.com/308610/103293747-75914280-49be-11eb-8520-9957c665ed0f.png)